app/assets/stylesheets/alchemy/errors.scss

```diff
@@ -37,3 +37,10 @@ div#errors, div.errors {
     margin-bottom: 0.5em;
   }
 }
+
+body.error {
+  .error.message {
+    max-height: 95%;
+    overflow-y: scroll;
+  }
+}
```

Comment:
I don't understand the need for the complication here to only apply the `error.message` class only when the body has a class of `error`? Ain't it too much?